web/pdf_presentation_mode.js

```diff
@@ -177,6 +168,32 @@ var PDFPresentationMode = (function PDFPresentationModeClosure() {
     /**
      * @private
      */
+    _goToPreviousPage: function PDFPresentationMode_goToPreviousPage() {
+      var page = this.pdfViewer.currentPageNumber;
+      // If we're at the first page, we don't need to do anything.
+      if (page === 1) {
```

Comment:
Here and at `if (page === this.pdfViewer.pagesCount) ` add less/greater-equal comparison: e.g. page <= 1